climpred/comparisons.py

```diff
@@ -232,7 +232,7 @@ def _e2r(ds, reference):
 
     """
     if 'member' in get_dims(ds):
-        print("Taking ensemble mean...")
+        #print("Taking ensemble mean...")
```

Comment:
was annoying to see on screen all the time when bootstrapping

Reply:
should remove if unnecessary; someday maybe implement logger.debug

Reply:
@bradyrx would you be OK with that removed?

Reply:
Yeah I am okay with that. If the user is expecting an ensemble mean comparison and submits something with members, they should expect that we take the ensemble mean.